release notes/v0.50.0.md

```diff
@@ -0,0 +1,171 @@
+k6 `v0.50.0` is here 🎉! 
+
+This release:
+- Adds support for uploading files from the browser module.
+- Introduces the `options.cloud` option.
+- Stabilizes the previously experimental timers module as the `k6/timers` module.
+- Brings JSON Web Key support to the `k6/experimental/webcrypto` module.
+
+## Breaking changes
+
+- [websockets#60](https://github.com/grafana/xk6-websockets/pull/60) allows manually setting the `name` tag, which also overwrites the `url` tag with the `name` value. This change makes it consistent with the logic that was implemented in k6 v0.41. Thanks, @mkadirtan for contributing!
+
+### Browser APIs to Async
+
+In future releases, we are going to be moving most of the synchronous browser APIs to asynchronous ones (promisifying them). We expect this will affect most of our users, so we are posting this upfront before making the change. Here are the reasons for making this large breaking change:
+
+1. Most browser APIs use some form of long-running IO operation (networking) to perform the requested action on the web browser against the website under test. We need to avoid blocking Javascript's runtime event loop for such operations.
```

Comment:
```suggestion
1. Most browser APIs use some form of long-running IO operation (networking) to perform the requested action on the web browser against the website under test. We need to avoid blocking JavaScript's runtime event loop for such operations.
```